lte/gateway/configs/spgw.yml

```diff
@@ -40,3 +40,6 @@ ovs_multi_tunnel: true
 
 # Non-NAT specific config
 ovs_uplink_port_number: 2
+
+# Flag to control ovs table=0 using pipelined
+pipelined_config_enabled: false
```

Comment:
IMO this is a bit unclear, pipelined_managed_tbl0 might be better.

Reply:
Comments addressed